com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/AbortProcessAction.java

```diff
@@ -19,6 +20,7 @@ public AbortProcessAction(FlowableFacade flowableFacade, List<AdditionalProcessA
 
     @Override
     public void executeActualProcessAction(String userId, String superProcessInstanceId) {
+        flowableFacade.setProcessVariable(superProcessInstanceId, Constants.VAR_USER, userId);
```

Comment:
Check if OptimisticLockingException is possible here.
We discussed the scenario in person.